TrackParamFilter/sections/specific.txt

```diff
@@ -6,6 +6,7 @@
 ! Bad: ||example.org^$xmlhttprequest,redirect=noopvast-3.0 (for instance, should be in specific.txt)
 !
 !
+||whale.naver.com^$removeparam=wpid
```

Comment:
please add a description of the parameter or the link of the issue

Reply:
Oh. I forgot to add.
Please just a sec.